src/hnsw/scan.c

```diff
@@ -191,13 +190,13 @@ bool ldb_amgettuple(IndexScanDesc scan, ScanDirection dir)
 
         if(scanstate->distances == NULL) {
             scanstate->distances = palloc(k * sizeof(float));
-            ;
         }
         if(scanstate->labels == NULL) {
             scanstate->labels = palloc(k * sizeof(usearch_label_t));
         }
 
         // hnsw_search(scanstate->hnsw, vec->x, k, &num_returned, scanstate->distances, scanstate->labels);
+        elog(INFO, "querying index for %d elements", k);
```

Comment:
Can you make sure this is only triggered for the tests you wrote and not for all the other tests as well?